### DISCUSSION TITLE
Is it possible to have collections showing up on each Library Recommended page?

### DISCUSSION THREAD

**Discussion by jl94x4** [lostb1t/replex#84] [Ideas]
Is it possible to do this?

For example, I have a Star Wars collection and they are all named the same across the libraries, but obviously items from say Kids TV (clone wars animated series) won't show up in Star Wars collection in my TV Libraries (Obi-Wan Kenobi, Mandalorian, etc).

Is it possible to get this behaviour, so each collection across all libraries that are named the same also show up in recommended library? 

I know it works on the home screen but I've always wanted Plex to add the possibility of cross library collections at library level. I love Replex and I think is this is do-able it would make Replex possibly the best tool available for Plex server owners.

**Comment by jl94x4** [comment]
@lostb1t

**Comment by Zorro92** [comment]
I'm confused about what you're trying to accomplish here

**Comment by lostb1t** [comment]
Plex is in a feature freeze, I will only apply fixes for the time being. MRs are always welcome.

Alto I really not sure what you are trieng to accomplish here. cross library within a library seems counter productive and confusing. That's what home is for.
